src/styles/units/_functions.scss

```diff
@@ -0,0 +1,3 @@
+@function calendarWidth ($size, $cells, $padding, $gap) {
```

Comment:
Function name should be a verb and describe what the function does. Consider renaming `calendarWidth` to `calculateCalendarWidth` or `getCalendarWidth`. 🤔